File: java/Cup_Cakes/LargestNumber.java
public class LargestNumber{

public static void main(String []args){

int[] arrNumber = {10, 9, 13, 2, 14};


int largest = arrNumber[0];

for(int index = 1; index < arrNumber.length; index++){
	if(arrNumber[index] > largest){
	largest = arrNumber[index];
	
	}
	
}

System.out.print(largest);

}

}

File: javaScript/Task/largestNumber.js
let number = 123459;
let largest = 0;

let num = number.toString();

for(let index = 0; index < num.length; index++){
	
	let output = parseInt(num[index]);
	
	if(output > largest){
	largest = output
	}
	
}

console.log(largest);
